src/Cli/InfoCliCommand.php

```diff
@@ -34,7 +34,7 @@ public function printDependencyVersions(OutputInterface $output, SymfonyStyle $s
             ['Dependency', 'Installed', 'Version'],
             [
                 ['rsync', $environmentUtility->isRsyncInstalled() ? '✔' : '✘', $environmentUtility->getRsyncVersion()],
-                ['mysql-client', $environmentUtility->isMysqlDumpInstalled() ? '✔' : '✘', $environmentUtility->getMysqlDumpVersion()['mysqlVersion']],
+                ['mysql-client', $environmentUtility->isMysqlDumpInstalled() ? '✔' : '✘', $environmentUtility->getMysqlDumpVersion()['mysqlVersion']], // name mysql-client is misleading. The version gettet is the Server Version for that client
```

Comment:
good point, we then should rename it to what it actually is to make sure there is no confusion that this is neither the "mysql-client-tool version" nor the "mysql-server version" or anything else but the "mysql-server version the client was built for"
- therefore i suggest renaming it to "mysql-distribution" (official name in the versioning) or "mysql-client-build-for", what do you think?

Reply:
I will name it `mysql-distribution`. We still have to rework the info command anyway.